<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>158</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgba(0,25,255,0.5);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="profile">
     <property name="styleSheet">
      <string notr="true">background: red;</string>
     </property>
     <property name="text">
      <string>Profile</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="sign_in">
     <property name="styleSheet">
      <string notr="true">background: red;</string>
     </property>
     <property name="text">
      <string>Sign in</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="sign_up">
     <property name="styleSheet">
      <string notr="true">background: red;</string>
     </property>
     <property name="text">
      <string>Sign up</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
